### ISSUE TITLE
Breadcrumbs: SVG text alternative and question

### ISSUE THREAD

**Issue by wlla** [NYPL/staff-picks#64]
One known fix needed and one question.

**Fix**
The SVG associated with the "Recommendations" link (`data-reactid="178"`) should have a visually hidden text alternative. I suggest "Return to". @courtneymcgee to confirm.

**Question**
This will follow our pattern from Discovery which includes the visually hidden text "you are here" right before the ordered list of pages. To fully match the pattern established in Discovery, the link would be in a `nav` region with `aria-label="Breadcrumbs"`.

**Comment by EdwinGuzman** [comment]
To follow up on the question: this is currently not set up to be a `nav` so the `aria-label="Breadcrumbs"` may not make sense, it's just a link. We can continue the pattern and make it a `nav` and list but does it make sense to have that when there will only be one item in the list?

**Comment by wlla** [comment]
Agreed, the `aria-label` should only be used if this switches to a `nav`. If we fully followed the pattern from discovery, we would add a second list item with the static text of the page title (given the list two items), remove the initial svg, and adjust a `span` of visually hidden text.
